packages/ui-text-area/src/TextArea/index.tsx

```diff
@@ -117,8 +117,8 @@ class TextArea extends Component<TextAreaProps> {
         if (this._highlightRef) {
           const entryStyle = window.getComputedStyle(entry.target)
           this._highlightRef.style.transition = 'none'
-          this._highlightRef.style.width = `calc(${entryStyle.width}px + 2.125rem)`
-          this._highlightRef.style.height = `calc(${entryStyle.height}px + 2.125rem)`
+          this._highlightRef.style.width = `calc(${entryStyle.width} + 0.5rem)`
```

Comment:
entryStyle.width is a string, which already has 'px' in it. I removed the unneccessary px string from the `calc()` function and added the proper rem amount.